### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by a1346054**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -21,7 +21,7 @@ needrestart 3.6
     - [Core] Fix verbose/verbosity confusion in needrestart.conf.
       (github pull request #197 by Jan-Philipp Litza @jplitza)
 
- -- Thomas Liske <thomas@fiasko-nw.net>
```
Comment (liske) [inline ChangeLog:24]: This is intentional, the date is appended during release.

**Review by liske — CHANGES_REQUESTED** [review]

File: ChangeLog (hunk 1 above)
Comment (a1346054) [reply ChangeLog:24]: Apologies, I thought I removed that part of the commit already, it slipped through.

**Review by liske — APPROVED** [review]

**Comment by liske** [comment]
Looks fine to me, if you would resolve intentional spaces in `ChangeLog` I would happy to merge it.

**Comment by liske** [comment]
Thanks!

**Pull request merged**